webservice/dispatcher.php

```diff
@@ -55,7 +62,7 @@
 } elseif (isset($_GET['ws_key'])) {
     $key = $_GET['ws_key'];
 } else {
-    header($_SERVER['SERVER_PROTOCOL'].' 401 Unauthorized');
+    header($_SERVER['SERVER_PROTOCOL'] . ' 401 Unauthorized');
```

Comment:
It seems that PHPCSFixer doesn't fix errors in `webservice` directory

Reply:
Do you want me to remove the space are not ?

Reply:
add spaces between dot please.

Reply:
I already add them in my PR that's why I ask the question